src/main/java/com/rackspace/ceres/app/services/TimestampProvider.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspace.ceres.app.services;
+
+import java.time.Instant;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TimestampProvider {
```

Comment:
I've not noticed where this is used but it seems a bit pointless?

Reply:
Oops, that wasn't needed in the end.